service/src/main/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelper.java

```diff
@@ -99,4 +101,29 @@ private static String mapJudgeOrAdvisor(JudgeAndLegalAdvisor judgeAndLegalAdviso
                 return judgeAndLegalAdvisor.getJudgeTitle().getLabel() + " " + judgeAndLegalAdvisor.getJudgeLastName();
         }
     }
+
+    public Optional<JudgeAndLegalAdvisor> buildForHearing(CaseData caseData,
```

Comment:
can this go above the private methods

Reply:
You want all the private methods that contribute to your public method to be in the same place. It follows the same principle of using/initialising the variable straight before the usage